channels/applicationinsights-channel-js/src/SendBuffer.ts

```diff
@@ -91,8 +97,20 @@ abstract class BaseSendBuffer {
 
                     return;
                 }
-
+         
```

Comment:
so get events from older buffer and add them back to new buffer  and "kills" the old buffer key